README.md

```diff
@@ -119,6 +119,9 @@ how to fix it.
   file\:* type `TAB`. (What this actually does is insert the currently
   selected candidate into the minibuffer, which for `find-file` has
   the effect of navigating into a directory.)
+* *To copy the current candidate:* type `M-w` or what is bind to
+  `kill-ring-save`. When there's an active region in your input, this
+  copies the active region.
```

Comment:
I'd like this description to make it clear that the default behavior of `M-w` is only modified when there is no active region and Transient Mark mode is disabled (so there is no reasonable action for `M-w` anyway).

Reply:
Done, but I'm not sure whether I made it clear. I didn't know Transient Mark mode until today.